release/7-3/ubi9/meta.json

```
@@ -0,0 +1,17 @@
+{
+    "IsLinux" : true,
+    "UseLinuxVersion": false,
+    "PackageFormat": "powershell${channelTag}-${PS_VERSION}-1.rh.x86_64.rpm",
+    "SkipGssNtlmSspTests": true,
+    "osVersion": "Universal Base Image 9",
+    "ShortDistroName": "ubi",
+    "shortTags": [
+        {"Tag": "9"}
+    ],
+    "SubImage": "test-deps",
+    "TestProperties": {
+        "size": 502
+    },
+    "DistributionState": "ImageCreation",
+    "EndOfLife": "2023-05-30"
```

I think this is 2024, but I'm unsure because they only have a chart.
```suggestion
    "EndOfLife": "2024-05-30"
```